source/blog/articles/2015-05-13-brussels.md

```diff
@@ -4,7 +4,7 @@ author: Cecile Veneziani
 page_title: "Brussels with Co-Station and Le Wagon Brussels"
 title: "Brussels (13 may - 16 may)"
 cover_picture: IMG_20150516_193022.jpg
-banner_picture: IMG_20150516_182237.jpg
+banner_picture: IMG_20150515_195033.jpg
```

Comment:
your SEO need love, use `-` for keyword separator

    banner_picture: lewagon-brussels-coworking.jpg